scss/_popover.scss

```diff
@@ -23,19 +23,22 @@
   &.bs-tether-element-attached-bottom {
     margin-top: -$popover-arrow-width;
 
-    .popover-arrow {
-      bottom: -$popover-arrow-outer-width;
+    &::before,
```

Comment:
@kkirsche Ah, I'm not watching #17052 and hadn't seen the subsequent comments where he gave in.
Nevermind then! :smile: